Library/Formula/websphere-liberty-profile.rb

```diff
@@ -0,0 +1,70 @@
+class WebsphereLibertyProfile < Formula
+  desc "WebSphere Liberty Profile"
+  homepage "https://developer.ibm.com/wasdev/websphere-liberty/"
+  url "https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/8.5.5.7/wlp-javaee7-8.5.5.7.zip"
+  sha256 "77f710c93cfd45661782d9895d5a17caa751a4186003753044e06b730d380bba"
+
+  depends_on :java => "1.7+"
+
+  def install
+    rm_rf Dir["bin/*.bat"]
+    rm_rf "usr"
+    rmdir "usr"
+
+    binaries = %w[binaryLog collective ddlGen installUtility securityUtility serverSchemaGen client configUtility featureManager isadc productInfo server]
```

Comment:
It'd be nice to split this onto two lines to make the formula more readable. You can still do that in the array.